notebooks/stress-tests.ipynb

```diff
@@ -38,8 +38,8 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        failed_StressTestResults = []
Line #152.       tressTestProgression:

</pre><p>Rename variables in lower camel case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>